src/version.ts

```diff
@@ -1 +1 @@
-export const version = '3.3.3';
\ No newline at end of file
+export const version = '3.3.3';
```

Comment:
Should this be updated to 3.3.4?

Reply:
🙄  yes, this is another project with the annoyance of having multiple places to look for bumping versions which is really redundant.

Reply:
Agreed, what is this even used for? 😥

Reply:
This was the published `package.json` and also referred to for the version which is put into the version file.

The next iteration of this (which will be a breaking change) I want to improve this repository and stop supporting both browser fetch as well as node axios. The project is meant to be for purely node wrapper and not browser support. The javascript wrapper does that.

Reply:
Sounds like a plan!